src/perf_mainloop.cc

```diff
@@ -31,7 +31,13 @@ extern "C" {
 
 #include "defer.hpp"
 
-#define rmb() __asm__ volatile("lfence" ::: "memory")
```

Comment:
I already replaced the memory barrier by atomic_load/acquire and atomic_store/release that should mimic what is provided by ring_buffer API (https://github.com/torvalds/linux/blob/v5.16/tools/include/linux/ring_buffer.h#L59).
But I forgot to remove rmb definition ;)

Reply:
Removed in latest commit